docs/get-started/how-it-works.mdx

```diff
@@ -30,46 +38,93 @@ A commons' data is stored on its users' devices using **pockets**.
+Documents can either belong to a single user, or be writable by everyone in the commons.  In that case many people can write to the same document, and your pocket will store the last version from each author. This means you can always roll back or resolve conflicts if something went wrong.
 
-Every document has a signature to help Earthstar verify its authorship. So if a document claims it was written by `@suzy`, you can be fairly confident it was really written by `@suzy`. These signatures are created using **identities**.
+Every document has a signature to help Earthstar verify its authorship. So if a document claims it was written by `@suzy`, you can be confident it was really written by `@suzy` and not alteredy by anyone else. These signatures are created using **identities**.
 
 ## Identities
 
-An identity in Earthstar is represented by a keypair, which is a bit like a username and password.
+A person's identity in Earthstar is represented by a keypair, which is a bit like a username and password.
 
-It's made up of an **address** and a **secret**. The address is made up of a shortname you choose and something called a public key, which can look like this: `@suzy.bzfkbawuj2cdxsxilvso4a2xczmcx6char4pqd2tjhdmcoy5i4ebq`.
+It's made up of an **address** and a **secret**. The address is made up of a 4-letter "shortname" you choose and something called a public key, which can look like this: `@suzy.bzfkbawuj2cdxsxilvso4a2xczmcx6char4pqd2tjhdmcoy5i4ebq`.
 
 The address is publicly shareable information, and the secret should be kept, well, secret. A keypair's address or secret can never be changed, so keep it safe!
 
-When a public name is put into a document's path, prefixed with a tilde (e.g. `/about/@suzy.bzfkbawuj2cdxsxilvso4a2xczmcx6char4pqd2tjhdmcoy5i4ebq/displayName.txt`), then _only_ the user with that identity can write to that document.
+When a public name is put into a document's path, prefixed with a tilde (e.g. `/about/~@suzy.bzfkbawuj2cdxsxilvso4a2xczmcx6char4pqd2tjhdmcoy5i4ebq/displayName.txt`), then _only_ the user with that identity can write to that document.  Otherwise anyone can edit the document.
 
 But what about the ways we express our identity, like names, pronouns, or avatars?
 
-This kind of information is stored as data within the commons you use. This has two big benefits: your identity can freely fluctuate between the spaces you use; and you can express your identity with as little or as much detail as you want!
+This kind of information is stored as documents within the commons, e.g. in your ["about me"](https://github.com/earthstar-project/earthstar/wiki/Standard-paths-and-data-formats-used-by-apps#about-author-profile-info) document. This has two big benefits: your identity can freely fluctuate between the spaces you use; and you can express your identity with as little or as much detail as you want!
```

Comment:
This is a section we really need moved to the website. It describes Earthstar's appeal so well, it's a shame to leave it in a github wiki.

Reply:
I put it in the wiki so that it would be easy for other people to document their formats, so consensus could emerge without being bottlenecked by us.  We could put some of the more important formats on the website also with a note to consult the wiki for the latest version

Reply:
I understand. Sounds good.